Apps/backend/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "backend",
-  "type":"module",
+  "type": "module",
```

Comment:
I find it very difficult to do a review when there are a lot of linter improvements in between. This should actually be in the scope of an extra pull request.